src/ui_contextmenu.cpp

```diff
@@ -338,6 +423,15 @@ class OpenLyricsContextSubItem : public contextmenu_item_simple
             case cmd_mark_instrumental:
                 out = "Remove existing lyrics and skip future automated lyric searches";
                 return true;
+            case cmd_seek_to_next_lyric:
+                out = "Seek to next lyrics timestamp";

```

Comment:
Ideally we should see if we can add more information here or give a slightly more detailed description of what the command/button does (and similarly for the other options). For example:

```suggestion
                out = "Seek to the timestamp of the next line of timestamped lyrics";
```